crates/viewer/re_renderer/src/importer/stl.rs

```diff
@@ -20,18 +20,17 @@ pub fn load_stl_from_buffer(
 ) -> Result<CpuModel, StlImportError> {
     re_tracing::profile_function!();
 
-    let cursor = std::io::Cursor::new(buffer);
-    let StlData {
-        name,
-        triangles,
-        normals,
-        ..
-    } = StlData::read_buffer(std::io::BufReader::new(cursor)).map_err(StlImportError::TinyStl)?;
+    let mut cursor = std::io::Cursor::new(buffer);
+    let IndexedMesh { vertices, faces } =
+        stl_io::read_stl(&mut cursor).map_err(StlImportError::StlIoError)?;
+    let num_vertices = faces.len() * 3;
 
-    let num_vertices = triangles.len() * 3;
+    // TODO(gijsd): parse name from ASCII?
+    // https://github.com/hmeyer/stl_io/pull/26
+    let name = "";
```

Comment:
bit of a regression. We can live with that but then let's patch the whole thing up to require a `DebugLabel` being passed in.